certmanager/ca.go

```diff
@@ -103,7 +103,9 @@ func (ca IstioCA) Generate(name, namepsace string) (chain, key []byte) {
 		SignerCert:   ca.signingCert,
 		SignerPriv:   ca.signingKey,
 		IsCA:         false,
+		IsClient:     true,
```

Comment:
I kind like keeping them - the CA cert doesn't need to set IsClient and IsServer. I may clean more things up in a followup PR.